### PR TITLE
fix: local nextauth

### PR DESCRIPTION
**Pull request by tomkit**
Fixes locally running development environment

## Test Plan

Logged in locally using next-auth

### CODE REVIEW

File: apps/mgmt-ui/src/pages/index.tsx [hunk 1]
```diff
@@ -1,34 +1,22 @@
...
 import { type GetServerSideProps } from 'next';
 import { getServerSession, Session } from 'next-auth';
-import { API_HOST, IS_CLOUD, ORGANIZATION_ID, SG_INTERNAL_TOKEN } from './api';
+import { API_HOST, ORGANIZATION_ID, SG_INTERNAL_TOKEN } from './api';
 
 export const getServerSideProps: GetServerSideProps = async ({ req, res }) => {
   let session: Session | null = null;
 
-  let orgId = ORGANIZATION_ID;
+  const orgId = ORGANIZATION_ID;
```
Comment (asdfryan) [inline apps/mgmt-ui/src/pages/index.tsx:9]: I think this will break it for cloud. Did you test with cloud = true?

File: apps/mgmt-ui/.env.sample [hunk 2]
```diff
@@ -0,0 +1,4 @@
+NEXTAUTH_URL=http://localhost:3000
+SUPAGLUE_JWT_SECRET=some-secret
+SUPAGLUE_INTERNAL_TOKEN=some-internal-token
+SUPAGLUE_API_HOST=http://localhost:8080
```
Comment (lucasmarshall) [inline apps/mgmt-ui/.env.sample:4]: We should add to the development docs that you need to copy this

**Review by asdfryan — CHANGES_REQUESTED** [review]
I think your change will break cloud, let me know if this isn't the case.

File: apps/mgmt-ui/src/pages/index.tsx (hunk 1 above)
Comment (tomkit) [reply apps/mgmt-ui/src/pages/index.tsx:9]: vscode did this -- it doesn't look like it's getting assigned anywhere in the code block

Comment (tomkit) [reply apps/mgmt-ui/src/pages/index.tsx]: I removed some code in test -- restored

File: DEVELOPMENT.md [hunk 3]
```diff
@@ -16,7 +16,9 @@ cp docker-compose.override.dev.yml docker-compose.override.yml
 ### Running
 
 ```shell
-docker-compose up
+cp apps/mgmt-ui/.env.sample apps/mgmt-ui/.env
```
Comment (lucasmarshall) [inline DEVELOPMENT.md:19]: Still need to run `docker compose up`

Comment (tomkit) [reply DEVELOPMENT.md:19]: This is to run the entire stack or just the FE

Comment (lucasmarshall) [reply DEVELOPMENT.md:19]: This is for running the full stack

**Review by lucasmarshall — APPROVED** [review]

**Pull request merged**